src/lib.rs

```diff
@@ -315,6 +339,10 @@ impl<'tcx> Evaluator<'tcx> {
     fn new(validate: bool) -> Self {
         Evaluator {
             env_vars: HashMap::default(),
+            mem_fds: HashMap::default(),
+            // start out with no stack root pointer by using a forever invalid AllocId
+            stack_addr: Pointer::new(AllocId(u64::max_value()), Size::ZERO).with_default_tag(),
```

Comment:
I used `Option` as initial value for `argc`, argv`, `cmd_line`. We should be consistent about this. I think I prefer `Option` because if we screw up it'll fail much more reasonably.